.github/workflows/ci2.yml

```diff
@@ -72,7 +72,7 @@ jobs:
           - distro: fedora
             base: centos.8
             with: fedora:35
-            java: no
+            java: yes
```

Comment:
Since this ends up getting evaluated by shell script, use `true`(/`false`) here as it lets you write more language natural shell script later.

Reply:
sounds good.